.sscignore

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Does this file have any impact?

Reply:
not at the minute, there are directions for handling an exception to ignore warnings that are not an actual problem here: https://eng.ms/docs/cloud-ai-platform/devdiv/one-engineering-system-1es/1es-docs/secure-supply-chain/project-artemis/central-feed-services-cfs, there is form which needs to be filled out which I have now submitted. Once it is looked at this should work to suppress the warnings.